pkg/docker-engine/Dockerfile

```diff
@@ -148,8 +148,16 @@ RUN --mount=type=bind,from=scripts,source=rpm-init.sh,target=/usr/local/bin/rpm-
     rpm-init $DISTRO_NAME
 COPY rpm /root/rpmbuild/SPECS
 ARG TARGETPLATFORM
-RUN --mount=type=bind,from=scripts,source=rpm-builddep.sh,target=/usr/local/bin/rpm-builddep \
-    rpm-builddep $(xx-info rhel-arch) /root/rpmbuild/SPECS
+RUN --mount=type=bind,from=scripts,source=rpm-builddep.sh,target=/usr/local/bin/rpm-builddep <<EOT
+  set -e
+  no_libnftables=0
+  case "$DISTRO_NAME" in
+    rhel*)
+      no_libnftables=1
```

Comment:
Maybe we should add a comment _why_ we're skipping it on RHEL (was thinking that later), because RHEL supports it, but it currently requires a subscription to install the required dependencies.


(And, as a further follow-up, we could even consider adding a way to override this, so that it _can_ be built if a subscription is in place)

Reply:
Added comment in https://github.com/docker/packaging/pull/284/files#diff-2eafab85eee78d035c1f1b46aec61fce6d76d11d3002428d91cf5d76eed8ee12R64-R67